statsmodels/regression/tests/test_mixed_glm.py

```diff
@@ -0,0 +1,197 @@
+        np.testing.assert_allclose(fp2, 0, atol=1e-3)
+
+        # Check the Hessian at the mode.
+        he1 = approx_hess(mp.ravel(), fun1)
+        he2 = approx_fprime(mp.ravel(), grad)
+        he3 = hess(mp.ravel())
+
+        np.testing.assert_allclose(he1, he2, atol=1e-4, rtol=1e-4)
+        np.testing.assert_allclose(np.linalg.slogdet(he2)[1], he3)
+
+        for k in range(10):
+
+            # An arbitrary state for the random effects
+            ref = np.random.normal(size=(model.n_groups, model.k_re)).ravel()
+
+            # Check that the two functions values are equal.
+            np.testing.assert_allclose(fun1(ref), fun2(ref))
+
+            # Check the gradient
+            fp1 = approx_fprime(ref.ravel(), fun1)
+            fp2 = grad(ref.ravel())
+            np.testing.assert_allclose(fp1, fp2, atol=1e-3)
+
+            # Check the Hessian
+            he1 = approx_hess(ref.ravel(), fun1)
+            he2 = approx_fprime(ref.ravel(), grad)
+            he3 = hess(ref.ravel())
+            np.testing.assert_allclose(he1, he2, atol=1e-4, rtol=1e-4)
+            np.testing.assert_allclose(np.linalg.slogdet(he2)[1], he3)
+
+
+def test_laplace_loglike_binomial():
+    """
+    Python:
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Binomial', 30)
+    mat = np.hstack((endog[:, None], groups[:, None], exog))
+    np.savetxt('mat.txt', mat)
+
+    R:
+    df = read.table('mat.txt')
+    names(df) = c('y', 'g', 'x1', 'x2')
+    library(lme4)
+    glmer(y ~ 0 + x1 + x2 + (1 | g), family=binomial(), data=df)
+
+    logLik -31.7866107
+    """
+
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Binomial', 30)
+
+    model = MixedGLM(endog, exog, groups=groups, family=sm.families.Binomial())
+    par = MixedGLMParams.from_components(fe_params=np.asarray([0.7400841, 1.279978]),
+                                         cov_re=np.asarray([[0.5818]]))
+    logLik = model.loglike(par)
+    np.testing.assert_allclose(logLik, -31.7866107)
+
+
+def test_laplace_loglike_poisson():
+    """
+    Python:
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Poisson', 30)
+    mat = np.hstack((endog[:, None], groups[:, None], exog))
+    np.savetxt('mat.txt', mat)
+
+    R:
+    df = read.table('mat.txt')
+    names(df) = c('y', 'g', 'x1', 'x2')
+    library(lme4)
+    glmer(y ~ 0 + x1 + x2 + (1 | g), family=poisson(), data=df)
+
+    logLik -85.9951
+    """
+
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Poisson', 30)
+
+    model = MixedGLM(endog, exog, groups=groups, family=sm.families.Poisson())
+    par = MixedGLMParams.from_components(fe_params=np.asarray([1.01505, 0.9464392]),
+                                         cov_re=np.asarray([[0.1195]]))
+    logLik = model.loglike(par)
+    np.testing.assert_allclose(logLik, -85.9951, rtol=1e-5)
+
+
+# Not working for non-canonical GLM links yet. 
+
+# def test_laplace_loglike_gamma():
```

Comment:
you can disable tests by changing the name instead of commenting out, that's less interuptive for the change history.
I often just add an underline like `t_est_laplace_loglike_gamma` or drop the "t" in "test", then nose will not identify as a unit tests